main/main.c

```diff
@@ -95,4 +48,11 @@ void app_main(void)
 
     rm_route("192.168.60.0", "255.255.254.0");
     print_route_table();
+    #ifdef CONFIG_RING_LINK_LOWLEVEL_IMPL_SPI
+    printf("CONFIG_RING_LINK_LOWLEVEL_IMPL_SPI\n");
```

Comment:
Where is defined?

Reply:
https://github.com/changeemma/esp32-internet-for-all/pull/5/files#diff-e3c09e822e4b80c1336fc48bd80280bcda21968f57ecb3cc41e09cda0252c56dR7-R11